apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/client/url/OpenApiPathBuilder.java

```diff
@@ -84,7 +84,13 @@ public OpenApiPathBuilder namespacesPathVal(String namespaces) {
   }
 
   public OpenApiPathBuilder itemsPathVal(String items) {
-    pathVariable.put(ITEMS_PATH, escapePath(items));
+    if(UrlUtils.hasIllegalChar(items)){
+      items = new String(Base64.getUrlEncoder().encode(items.getBytes(StandardCharsets.UTF_8)));
```

Comment:
How about extracting this duplication of `Base64.getUrlEncoder().encode(items.getBytes(StandardCharsets.UTF_8))` into UrlUtils?